rc/filetype/markdown.kak

```diff
@@ -30,6 +30,15 @@ hook -group markdown-highlight global WinSetOption filetype=markdown %{
     hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/markdown }
 }
 
+declare-option \
```

Comment:
This could still be in the shell block (using the map syntax). I am not really keen on that option because its only when the module is loaded that it gets used, so changing it after will not work and could be confusing.

Reply:
I agree that this is more a pragmatic rather than an elegant solution. I was looking for a way to allow third-party highlighters in code blocks. 

However, I will adjust the code to at least allow for the second use case, i.e., to allow synonyms for languages.

Reply:
You can add a group in `markdown`.

When option changes, build the group.

Reply:
Now, in the summer holidays I have taken another shot at this., see the new forced push